### PR TITLE
Support prerelease from main instead of prerelease

### PR DESCRIPTION
**Pull request by dibarbet**
Example - https://dnceng.visualstudio.com/internal/_build/results?buildId=2241794&view=results

Now you have to manually select the build to release

### CODE REVIEW

**Review by arkalyanms — APPROVED** [review]

File: azure-pipelines/release.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 trigger: none
 pr: none
 
+resources:
```
Comment (Cosifne) [inline azure-pipelines/release.yml:4]: Seems like you are suppose to create release from 'release' branch, instead always from main, is that correct?

Comment (dibarbet) [reply azure-pipelines/release.yml:4]: This is just sets the default branch to pick from, you can actually pick any branch and build you want in the resources screen.

Running this on a main build will upload a prerelease, running on a release branch build will upload a release

Comment (dibarbet) [reply azure-pipelines/release.yml:4]: image

Comment (Cosifne) [reply azure-pipelines/release.yml:4]: Yeah I am fine with it but just want to check (because now we run everything from main)

Comment (dibarbet) [reply azure-pipelines/release.yml:4]: oh sorry I misunderstood - no the pipeline yaml comes from main still.  Even if the pipeline branch is main you can pick a resource from any branch

**Pull request merged**